src/Microsoft.Performance.Toolkit.Engine.Tests/ToolkitEngineCreateInfoTests.cs

```diff
@@ -18,5 +24,75 @@ public void DefaultRuntime_Set()
             string expectedName = typeof(EngineCreateInfo).Assembly.GetName().Name;
             Assert.AreEqual(expectedName, sut.RuntimeName);
         }
+
+        [TestMethod]
+        [IntegrationTest]
+        public void DefaultProcessorOptionsResolver_Set()
+        {
+            var sut = new EngineCreateInfo(DataSourceSet.Create().AsReadOnly());
+            Assert.IsNotNull(sut.OptionsResolver, "Options Resolver is null when a default is expected");
+            Assert.AreEqual(typeof(GlobalProcessingOptionsResolver), sut.OptionsResolver.GetType(), "Options Resolver is not of type GlobalProcessingOptionsResolver");
```

Comment:
This is very glass-box test that's asserting implementation details instead of the spec.

Perhaps a more robust way to check this is to verify that `sut.OptionsResolver.GetProcessorOptions` returns `ProcessorOptions.Default`? Since that field is static, you can do reference equality on it against the returned options